enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -88,6 +90,61 @@ <h3 class="course-confirmation-title">${ confirmation_text }</h3>
         </div>
       </div>
     </div>
-
+    <div id="course-details-modal" class="modal" aria-hidden="true">
+      <div id="course-details-modal-content" class="modal-content" role="dialog" aria-modal="true" aria-labelledby="modal-header-text">
+        <button class="modal-close-button" id="modal-close-button" aria-label="${_('Close')}">${_('Close')}</button>
+        <header class="wrapper">
+          <div class="image">
+            <img src="${ course_image_uri }" alt="${ course_name }"/>
+          </div>
+          <div class="details">
+            <h1 id="modal-header-text">${ course_name }</h1>
+            <p class="short-description">${ course_short_description }</p>
+            <div class="enterprise">
+              <img src="${static.url('images/logo.png')}" alt="${ course_organization }" />
```

Comment:
@haikuginger Is there any disadvantages with using the https://github.com/edx/edx-platform/blob/master/common/djangoapps/util/organizations_helpers.py module instead of the URL approach?